packages/jest-util/src/shuffleArray.ts

```diff
@@ -0,0 +1,38 @@
+// Mulberry 32 taken from
```

Comment:
is there some node module we can use instead of rolling our own?

Reply:
I've replaced it with a package called Prando https://www.npmjs.com/package/prando
Though overly complicated for our needs it's pretty popular and has 0 dependencies.